extensions/common/api/api-observability/src/main/java/org/eclipse/edc/api/observability/ObservabilityApiExtension.java

```diff
@@ -33,7 +33,7 @@ public class ObservabilityApiExtension implements ServiceExtension {
 
     public static final String NAME = "Observability API";
     public static final String OBSERVABILITY_CONTEXT = "observability";
-    private static final String API_VERSION_JSON_FILE = "version.json";
+    private static final String API_VERSION_JSON_FILE = "obs-api-version.json";
```

Comment:
maybe it's better to use the term without abbreviations for consistency, same for the others